stack.yaml

```diff
@@ -1,18 +1,14 @@
 # For more information, see: https://github.com/commercialhaskell/stack/blob/release/doc/yaml_configuration.md
 
 # Specifies the GHC version and set of packages available (e.g., lts-3.5, nightly-2015-09-21, ghc-7.10.2)
-resolver: lts-5.0
+resolver: nightly-2017-01-14
```

Comment:
That job [already exists](https://github.com/fmthoma/vgrep/blob/master/.travis.yml#L27) although it's currently in the allow_failures section. Would you like to make it a required success?